File: small_problems/easy4/ex5.rb
def multisum(num)
  arr = []

  (1..num).each do |x|
    if x % 3 == 0 || x % 5 == 0
      arr << x
    end
  end
  arr.sum
end

p multisum(3) == 3
p multisum(5) == 8
p multisum(10) == 33
p multisum(1000) == 234168

File: exercises/js210_small_problems/easy_1/ex5.js
const readLine = require('readline-sync');

let integer = Number(readLine.question('Please enter an integer greater than 0: '));
let sumOrProduct = readLine.question('Enter "s" to compute the sum, or "p" to compute the product: ');

let answer = []
for (let i = 1; i <= integer; i += 1) {
  answer.push(i);
}

if (sumOrProduct === 's') {
  answer = answer.reduce((sum, num) => sum + num);
  sumOrProduct = 'sum';
} else {
  answer = answer.reduce((prod, num) => prod * num);
  sumOrProduct = 'product';
}

console.log(`The ${sumOrProduct} of the integers between 1 and ${integer} is ${answer}.`);
